velox/functions/prestosql/ArrayContains.cpp

```diff
@@ -112,7 +114,9 @@ void applyComplexType(
             searchBase,
             elementIndices[offset + i],
             searchIndex,
-            CompareFlags::NullHandlingMode::kNullAsIndeterminate);
+            throwOnNestedNull
+                ? CompareFlags::NullHandlingMode::kNullAsIndeterminate
+                : CompareFlags::NullHandlingMode::kNullAsValue);
```

Comment:
One concern here - we add this condition per row per element.
That should have some effect on performance.
And this change is needed for testing in fuzzer.

I realize that we want to reuse the code, however have we considered a less intrusive change, like copy/pasting (I know, sounds terrible) the array_contains code?

Reply:
Good catch! We actually don't need to check this condition per row per element even if we reuse the code, because throwOnNestedNull is a constant value to the loop. I've lifted it out of the loop in the update. Please let me know if you still have concerns about the performance. We can make throwOnNestedNull a template argument in that case to reduce the cost even further. (I would consider copy/pasting as the last option.)